contracts/VotingEscrow.vy

```diff
@@ -433,7 +440,6 @@ def deposit_for(_addr: address, _value: uint256):
     @param _addr User's wallet address
     @param _value Amount to add to user's lock
     """
-    assert(self.unlocked == False) # dev: no more lock
```

Comment:
Question:

Shouldn't add `self.migrated` here and [here](https://github.com/yearn/veYFI/pull/104/files#diff-244a97f574a321ee15a9bbcde95f07842637249f93f58dec8c39f6587efd272dL453)?